src/mixins/filing-template-mixin.ts

```diff
@@ -414,6 +415,9 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     // restore Business Number
     this.setRegistrationBusinessNumber(draftFiling.registration.business.taxId || null)
 
+    // restore Business Type Confirm
+    this.setRegistrationBusinessTypeConfirm(draftFiling.registration.businessTypeConfirm || null)
+
```

Comment:
This code assigns null to the store value if the draft filing doesn't have the property (for example, in a new draft). I'm thinking assigning false would make more sense. What do you think?

Reply:
I think setting it to false makes sense, so that when draftFiling.registration.businessTypeConfirm is null (or false), then it is false. It is kind of consistent with state-model initial value and readonly businessTypeConfirm!: boolean (line 40) in BusinessTypeConfirm.vue.

Reply:
I changed it to || false in my latest commit: c77ff41.